### PR TITLE
scaling pixel offset by distance for billboards from KML files

### PR DESCRIPTION
**Pull request by theOgrable**

### CODE REVIEW

**Comment by hpinkos** [comment]
Great, thanks @theOgrable!  Can you please send us a CLA?  See the [Contributing Guide](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#contributor-license-agreement-cla) for information on how to do this.

**Comment by pjcozzi** [comment]
We now have a CLA from @theOgrable, thanks!

File: Source/DataSources/KmlDataSource.js [hunk 1]
```diff
@@ -663,7 +669,7 @@ define([
         }
 
         //GE treats a heading of zero as no heading
-        //Yes, this means it's impossible to actually point north in KML
+        //You can still point north using a 360 degree angle (or any multiple of 360)
```
Comment (hpinkos) [inline Source/DataSources/KmlDataSource.js]: :+1:

**Comment by hpinkos** [comment]
I can confirm this fixes the problem.  @bagnell, does this change look good to you?

**Comment by hpinkos** [comment]
Looks good, thanks @theOgrable!

**Comment by pjcozzi** [comment]
@hpinkos update CHANGES.md in master if it is warranted.

**Pull request merged**